src/Microsoft.Framework.CopyOnWriteDictionary.Internal/Microsoft.Framework.CopyOnWriteDictionary.Internal.kproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="__ToolsVersion__" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
That's a pretty awesome kproj utility, have to admit :smile: I end up spending a bit of my time prior to any commits trying to manually clean up the kproj :+1: